examples/multigpu/graphbolt/node_classification.py

```diff
@@ -134,16 +134,14 @@ def create_dataloader(
     ############################################################################
     if args.storage_device != "cpu":
         datapipe = datapipe.copy_to(device)
-    datapipe = datapipe.sample_neighbor(graph, args.fanout)
+    datapipe = datapipe.sample_neighbor(
+        graph, args.fanout, overlap_fetch=args.storage_device == "pinned"
```

Comment:
Did we release the overlap_fetch in last release?

Reply:
I think it is in the DGL 2.3. In this release, this parameter moved from `gb.DataLoader` to `sample_neighbor`.

Reply:
https://docs.dgl.ai/en/latest/generated/dgl.graphbolt.DataLoader.html#dgl.graphbolt.DataLoader shows the overlap_graph_fetch option.